HDF4Examples/C/AN/h4ex_AN_get_annotation_info.c

```diff
@@ -8,130 +8,102 @@ main()
 {
     /************************* Variable declaration **************************/
 
-    intn  status_n;                     /* returned status for functions returning an intn */
-    int32 status_32,                    /* returned status for functions returning an int32*/
-        file_id, an_id, n_annots,       /* number of annotations */
-        *ann_list,                      /* list of annotation identifiers */
-        vgroup_ref,                     /* reference number of the vgroup */
-        index;                          /* index of an annotation in the annotation list */
+    int32    file_id, an_id;
+    int32    n_annots;                  /* number of annotations */
+    int32   *ann_list = NULL;           /* list of annotation identifiers */
+    int32    vgroup_ref;                /* reference number of the vgroup */
+    int32    index;                     /* index of an annotation in the annotation list */
     ann_type annot_type = AN_DATA_DESC; /* annotation to be obtained*/
     uint16   ann_tag, ann_ref,          /* tag/ref number of an annotation */
         vgroup_tag = DFTAG_VG;          /* tag of the vgroup */
-    int ret_val    = FAIL;
 
     /********************** End of variable declaration **********************/
 
     /*
-     * Open the HDF file.
+     * Create the HDF file.
```

Comment:
This is an open, not crThis is an open, not create. Am I missing something?eate. Am I missing something?

Reply:
No probably was right then the merge from library messed it up.